### PR TITLE
call restorecon on ssh host keys

### PR DESCRIPTION
**Pull request by hopkiw**

### CODE REVIEW

File: google_guest_agent/instance_setup.go [hunk 1]
```diff
@@ -263,6 +263,7 @@ func generateSSHKeys() error {
 			logger.Warningf("Generated key is malformed, not uploading")
 		}
 	}
+	runCmd(exec.Command("restorecon", "-FR", hostKeyDir))
```
Comment (zmarano) [inline google_guest_agent/instance_setup.go:266]: Probably should check for its existence this like we do here: https://github.com/GoogleCloudPlatform/guest-agent/blob/a0da4b0f3d552272d53101ffb0d65151a07fcd50/google_guest_agent/oslogin.go#L308

Comment (hopkiw) [reply google_guest_agent/instance_setup.go:266]: the reverse, actually - we don't care whether it succeeded or failed, so we just 'try' blindly. path check can be removed there. see also restorecon invocation in `non_windows_accounts.go`

Comment (zmarano) [reply google_guest_agent/instance_setup.go:266]: Oh ok then lets make it consistent everywhere :)

**Pull request merged**